Dockerfiles/Dockerfile-ubuntu-22.04

```diff
@@ -0,0 +1,32 @@
+FROM ubuntu:22.04
+# Note: kern PPA has been temporarily disabled until kernsuite provide 'jammy' packages. 
+# Until then, the 'casacore-data' package may not be fully up-to-date.
+
+# Install the basic packages
+RUN \
+  apt-get update && \
+  apt-get -y upgrade && \
+  DEBIAN_FRONTEND=noninteractive \
+  apt-get install -y apt-utils autoconf bison build-essential cmake curl fftw3-dev flex gcc g++ \
+    gdb gfortran git git-lfs libgmock-dev googletest libblas-dev libcfitsio-dev \
+    libgsl-dev libgtest-dev libhdf5-dev liblapack-dev libncurses-dev \
+    libprotobuf-dev libpugixml-dev libreadline-dev libssl-dev libstarlink-ast-dev \
+    libtool libxml2-dev libxslt1-dev libzstd-dev pkg-config protobuf-compiler \
+    python3-numpy python3-astropy software-properties-common unzip wcslib-dev wget uuid-dev
+
+# Get carta dependencies carta-casacore, fits2idia, and zfp from cartavis-team PPA and casacore-data from KERN
+RUN \
+  #apt-add-repository -y -s ppa:kernsuite/kern-7 && \
```

Comment:
I would completely remove these references to KERN from here -- the note at the top is sufficient. The KERN PPA targetting Jammy will have a different release version, so there's no reason to keep KERN-7 commented out here.